lib/desired-caps.js

```diff
@@ -34,6 +34,9 @@ let espressoCapConstraints = {
   },
   intentOptions: {
     isObject: true
+  },
+  displayId: {
```

Comment:
How about `launchDisplayId` or `activityOptionLaunchDisplayId` etc to make the naming more clear?

Reply:
yes. launchDisplayId is more clear.

Reply:
Here also should be `launchDisplayId`

Reply:
Yes. Fixed